tools/api-markdown-documenter/src/renderers/markdown-renderer/default-renderers/RenderSpan.ts

```diff
@@ -22,20 +21,6 @@ export function renderSpan(
 	node: SpanNode,
 	writer: DocumentWriter,
 	context: RenderContext,
-): void {
-	// Markdown tables do not support multi-line Markdown content.
-	// If we encounter a multi-line span in a table context, we will render using HTML syntax.
-	if (!node.singleLine && context.insideTable === true) {
```

Comment:
Spans only support PhrasingContent now, so this is not needed, and was resulting in falling back to HTML in scenarios that didn't need to (see the updated test below).